### DISCUSSION TITLE
Listing of working and non-working hardware configuration

### DISCUSSION THREAD

**Discussion by defdefred** [VendelinSlezak/BleskOS#24] [Ideas]
Hello Vendelín,
Very impressive work!

I tried several real computosaure I have and all worked.

How about making a listing of working and non-working hardware configuration ?

**Comment by VendelinSlezak** [comment]
At first, thank you for testing BleskOS on real hardware :)
List of working hardware is good idea, I was thinking about this a while ago but did not make it. But now, I will surely take a look and write this down. I personally have 10+ computers that I can use for testing BleskOS so it will take some time to collect all data.
Would you mind to provide some info about hardware you tested BleskOS so I will be able to also add it into list? Also, if it would be possible, I would greatly appreciate if you would post some photos from System log (Main window>[F1] System board>[L] System log) There are informations that I can use to identify hardware that is working and also catch unworking hardware.

**Reply by defdefred** [reply]
Sure I will do!
Please add a way to backup the System log to disk :-)

**Reply by VendelinSlezak** [reply]
I created page https://github.com/VendelinSlezak/BleskOS/wiki/List-of-working-hardware with some initial content.

"Please add a way to backup the System log to disk :-)"
That's really good idea. I will add it probably this Saturday.

**Comment by defdefred** [comment]
Nice!

Is booting iso from Ventoy Ok, or should I try img directly from usb too?

**Comment by VendelinSlezak** [comment]
Yes, you can also use Ventoy, after choosing bleskos iso you have to select option "Boot in memdisk mode" and it will work.
Update 11 have exporting System log as TXT file. You have to use USB formatted with FAT filesystem, because it is only read-write filesystem that BleskOS have driver for. And I highly recommend to use empty USB. Because even through from my tests BleskOS should work good, it is still in alpha state so it is better not to risk any data.

**Comment by defdefred** [comment]
Name: Dell vostro 1700 (https://mobilespecs.net/laptop/DELL/DELL_Vostro_1700.html)
Boot: USB ventoy BleskOS iso
Wav: This file cannot be played on this hardware
Notepad Mouse: ok
Notepad keyboard: ok
USB Mouse: ok (through USB hub)
USB keyboard ok but some stroke missing when fast typing (through USB hub)
Ethernet: ko
Backup System log: Error during rewriting folder. A strange folder "System Volume Information" is created instead of the specified filename.

```
ls -l /mnt
total 4776
-rwxr-xr-x 1 root root  181969 12 dec.  23:56 1plasma_energy.wav
-rwxr-xr-x 1 root root 1835008 16 dec.  18:23 bleskos.iso
-rwxr-xr-x 1 root root 2768882 12 dec.  23:56 image-de-paysage.jpg
-rwxr-xr-x 1 root root   96013 13 dec.  00:01 Naruto-PNG.png
drwxr-xr-x 2 root root    4096 16 dec.  19:47 System Volume Information

ls -l /mnt/System\ Volume\ Information/ 
total 8
-rwxr-xr-x 1 root root 76 16 dec.  19:47 IndexerVolumeGuid
-rwxr-xr-x 1 root root 12 16 dec.  19:47 WPSettings.dat

cat /mnt/System\ Volume\ Information/IndexerVolumeGuid | od -a
0000000   { nul   6 nul   6 nul   F nul   3 nul   2 nul   4 nul   5 nul
0000020   B nul   - nul   4 nul   B nul   B nul   1 nul   - nul   4 nul
0000040   9 nul   B nul   F nul   - nul   8 nul   1 nul   5 nul   E nul
0000060   - nul   4 nul   1 nul   A nul   5 nul   3 nul   3 nul   F nul
0000100   0 nul   B nul   4 nul   3 nul   D nul   } nul
0000114

cat /mnt/System\ Volume\ Information/WPSettings.dat | od -a
0000000  ff nul nul nul   H   S  ff   O   . nak   b   7
0000014
```

**Comment by VendelinSlezak** [comment]
I definitely need to see content of System log. There should be messages about what happend.

> Wav: This file cannot be played on this hardware

Root of this is probably in HDA driver. I improved HDA in update 12, so please try this again and we will see if it helped.

> Backup System log: Error during rewriting folder.

At first I would recommend you to even if it on first try show error, to try it one more time. If it do not help, it looks to me that you tried to save file to Ventoy USB, so you can try to use some other USB. But if it still do not save, please post content of System log as images. There should be error messages so I would be able to move forward on this issue.

> A strange folder "System Volume Information" is created instead of the specified filename.

BleskOS did not created this folder, it was Windows that automatically creates this folder on every connected USB and store some data in it.

**Comment by defdefred** [comment]
[vostro.txt](https://github.com/VendelinSlezak/BleskOS/files/13720983/vostro.txt)

I have been unable to use several fat formatted usb keys, but it worked with an usb flash drive.

wav still KO

**Comment by defdefred** [comment]
Name: Packard Bell iStart MC 5417
Wav: This file cannot be played on this hardware
USB Mouse: ok
USB keyboard ok
Ethernet: ok, but failed to get IP from DHCP
[packardbellmc5417.txt](https://github.com/VendelinSlezak/BleskOS/files/13732572/packardbellmc5417.txt)

**Comment by VendelinSlezak** [comment]
Thank you for this info, based on this logs I made several changes in Update 13, especially about HDA driver and USB mass storage devices, so you can try it and we will see if something changes.

> Name: Packard Bell iStart MC 5417
Wav: This file cannot be played on this hardware

I did not change AC97 driver, so this issue on this computer is not solved. Would it be please possible for you to send me your wav file, so I can examine it?

> Ethernet: ok, but failed to get IP from DHCP

Would it be possible for you to catch sended packet? Even when you connect two computers, BleskOS will automatically send DHCP Discover packet, so on second computer it is possible to see it for example in Wireshark. In that way we can find if some packet is even sended and if yes, what error does it have.

**Comment by defdefred** [comment]
[packardbellmc5417_u13.txt](https://github.com/VendelinSlezak/BleskOS/files/13746379/packardbellmc5417_u13.txt)
[infowav.txt](https://github.com/VendelinSlezak/BleskOS/files/13746410/infowav.txt)
[1plasma_energy.wav.gz](https://github.com/VendelinSlezak/BleskOS/files/13746412/1plasma_energy.wav.gz)

**Comment by defdefred** [comment]
Oppps missing tcpdump trace for the dhcp request...
[tcpdump.txt](https://github.com/VendelinSlezak/BleskOS/files/13756758/tcpdump.txt)

**Comment by VendelinSlezak** [comment]
Thank you for info. I found out you were unable to play WAV because it have more uncommon type. I wrote code for handling almost all types of WAV files, so your WAV can be played now. I also tested it on many real computers so I hope it will work also on yours computers. I also fixed one thing in USB msd, so I hope it will help your USB flash drives.

**Comment by defdefred** [comment]
Will re-create a discussion for each hardware of this thread.
